feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -31,8 +69,18 @@ export class ConfiguratorPriceComponent {
     if (this.formula.priceTotal) {
       return this.priceTotal;
     } else if ((this.formula.price?.value ?? 0) > 0) {
-      return '+' + this.formula.price?.formattedValue;
+      if (this.isRTLDirection()) {
+        return this.addSing(this.formula.price?.formattedValue, '+', false);
+      }
+      return this.addSing(this.formula.price?.formattedValue, '+', true);
```

Comment:
```suggestion
        return this.addSing(this.formula.price?.formattedValue, '+', !isRTLDirection);
```

Reply:
changed to: 
return this.addSing(this.formula.price?.formattedValue, '+', !this.isRTLDirection());